packages/orbit-components/src/InputSelect/types.d.ts

```diff
@@ -19,7 +19,7 @@ type KeyboardEvent = Common.Event<React.KeyboardEvent<HTMLInputElement>>;
 
 export interface Props extends Common.Globals, Common.SpaceAfter, Common.DataAttrs {
   readonly name?: string;
-  readonly label?: Common.Translation;
+  readonly label?: string;
```

Comment:
Just for the record, affected occurrences:

https://sourcebot.skypicker.com/search?query=%3CInputSelect%5B%5Cs%5CS%5D*%3Flabel%3D%7B(.*)%20-repo%3A%5Egithub%5C.com%2Fkiwicom%2Forbit%24